sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -59,6 +59,16 @@ public class AzureMonitorOptions : ClientOptions
         /// </summary>
         public string StorageDirectory { get; set; }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureMonitorOptions"/>.
+        /// </summary>
+        public AzureMonitorOptions()
+        {
+            // users can explicitly change it, but by default we don't want self-diagnostics to be reported to Azure Monitor.
```

Comment:
nit suggestion: Use 'internal logs' instead of self diagnostics to avoid potential confusion with https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#self-diagnostics though both are internal logs!